pyaedt/application/Analysis.py

```diff
@@ -1738,7 +1738,7 @@ def analyze_setup(
             except:
                 if set_custom_dso:
                     self.set_registry_key(r"Desktop/ActiveDSOConfigurations/" + self.design_type, active_config)
-                self.logger.error("Error in Solving Setup %s", name)
+                self.logger.error("Error in Solving All Setups")
```

Comment:
Error messages should be written as short but complete sentences: Error in solving all setups.